Create ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md (#1927)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-20-0-extend-the-dynamic-forms-page-in-codex.md

---
date: ""

> title: Extend the "Dynamic forms" page in Codex

Author: The Claude AI suggested to use the following title:
```suggestion
title: Dynamic forms page in Codex extended with additional information
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3282
version: 1019.20.0
---

> Extend the "Dynamic forms" page in Codex

Author: The Claude AI suggested to use the following body:
```suggestion
The Dynamic forms page in Codex lacked some important information to fully understand the feature. The page has been extended with additional details and examples to better explain the concept and usage of dynamic forms. This change makes it easier for users to comprehend how to utilize dynamic forms in their applications and provides a better reference for this feature in the documentation.
```

> Extend the "Dynamic forms" page in Codex

Reviewer: ```suggestion
The Dynamic forms page in Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.
```

> The Dynamic forms page in Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.

Reviewer 2: @bozhidardedov Is this the exact name? Can´t find it, can you please provide the path?

Reviewer: Yup, it's under Components > Forms > Dynamic forms.

> The Dynamic forms page in Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.

Reviewer 2: ```suggestion
The Components > Forms > Dynamic forms page in the Codex has been extended with additional details and examples to better explain the concept and usage of dynamic forms.
```
